pkg/auth/rhsso_authenticator.go

```diff
@@ -312,7 +312,6 @@ func (a *RHSSOAuthenticator) CreateAuthenticator() func(name, in string, authent
 				return true, nil, common.NewApiError(http.StatusInternalServerError, err)
 			}
 			if !ownedBy {
-				log.Errorf("Unauthorized access to cluster %s by a user other than the owner\n", clusterID)
```

Comment:
Maybe worth just reducing it to a debug log, and add a comment that it might flood the logs otherwise.
Also, a warning log in line 274 might be helpful.

Reply:
> Also, a warning log in line 274 might be helpful

This will just move the problem to somewhere else since we currently not differentiating between access to non-existing clusters and unauthorized access.

> Maybe worth just reducing it to a debug log

But then we will just flood the debug logs, how does that solve the problem.

Please see @avishayt 's comment https://coreos.slack.com/archives/CUPJTHQ5P/p1625663070173400?thread_ts=1625642128.124500&cid=CUPJTHQ5P.

@danielerez

Reply:
ok got it, so we would just eliminate the log for now to mitigate the issue.